src/BizHawk.Client.Common/movie/bk2/Bk2Movie.HeaderApi.cs

```diff
@@ -80,9 +70,8 @@ public bool StartsFromSaveRam
 			}
 		}
 
-		public string GameName
+		public override string GameName
 		{
-			get => Header.TryGetValue(HeaderKeys.GameName, out var s) ? s : string.Empty;

```

Comment:
This isn't the same implementation as the base. Is it wise to only override one?

Reply:
It should be, if you look at the definition of `Bk2Header`s `this[string key]` implementation: 
```csharp
	public class Bk2Header : Dictionary<string, string>
	{
		public new string this[string key]
		{
			get => TryGetValue(key, out var s) ? s : string.Empty;
			set => base[key] = value;
		}
```